module/VuFind/src/VuFind/Search/Base/Options.php

```diff
@@ -929,6 +938,16 @@ public function getRetainFilterSetting()
         return $this->retainFiltersByDefault;
     }
 
+    /**
+     * Should the "Reset Filters" button be displayed?
+     *
+     * @return bool
+     */
+    public function shouldDisplayResetFilters()
+    {
+        return $this->retainFiltersByDefault || $this->alwaysDisplayResetFilters;
```

Comment:
Would it be better to call getRetainFilterSetting here instead of directly accessing the property in case something is overridden in a custom subclass? (Not likely, but the abstraction seems harmless and potentially helpful).

Reply:
Yes, changed.